### ISSUE TITLE
 'NotebookApp' has no attribute 'load_jupyter_server_extension'

### ISSUE THREAD

**Issue by oeway** [Zsailer/nbclassic#9]
Hi, I am trying nbclassic but it seems broken.

Here is the command I used:
```
conda create -n jupyter-server python=3.6.8
conda activate jupyter-server
pip install nbclassic
jupyter server
```

Here is a part of the stack trace:
```
(jupyter-server) macbook$ jupyter server
Traceback (most recent call last):
...
  File "/Users/me/miniconda3/envs/jupyter-server/lib/python3.6/site-packages/traitlets/utils/importstring.py", line 42, in import_item
    return __import__(parts[0])
  File "/Users/me/miniconda3/envs/jupyter-server/lib/python3.6/site-packages/nbclassic/__init__.py", line 8, in <module>
    load_jupyter_server_extension = NotebookApp.load_jupyter_server_extension 
AttributeError: type object 'NotebookApp' has no attribute 'load_jupyter_server_extension'
```

I also tried `pip install jupyter_server nbclassic notebook` within in my base conda env, and get a different error message: `ImportError: cannot import name 'secure_write' from 'jupyter_core.paths' (/Users/me/miniconda3/lib/python3.7/site-packages/jupyter_core/paths.py)`

**Comment by Zsailer** [comment]
Oops. Looks like the nbclassic on PyPI needs a release. I think if you install nbclassic from source it will work for now. I’ll work on putting out a release in the next day or two.

**Comment by oeway** [comment]
@Zsailer Thanks! I confirm that the source installation worked.

**Comment by Zsailer** [comment]
Released version v0.1 today!
